doc/getting_started/installation_linux.rst

```diff
@@ -55,16 +55,17 @@ Install the required packages in a Ubuntu host system with:
 .. code-block:: console
 
    $ sudo apt-get install git make gcc g++ python3-ply ncurses-dev \
-	 python3-yaml dfu-util device-tree-compiler
-
+	 python3-yaml dfu-util device-tree-compiler python3-pip
+   $ pip3 install --user pykwalify
+   
```

Comment:
ah, missed amending that file after I corrected it -- ok, now it has the right one.